.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,76 @@
+name: docker-release
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - Dockerfile
+      - docker.Makefile
+  push:
+    branches:
+      - master
+      - main
+      - release/*
+      - landchecks/*
+    tags:
+      # Release candidate tags look like: v1.11.0-rc1
+      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+      - ciflow/nightly/*
+    paths:
+      - Dockerfile
+      - docker.Makefile
+  schedule:
+    - cron: 0 0 * * *
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+  cancel-in-progress: true
+
+env:
+  BUILD_PROGRESS: plain
+  BUILD_TYPE: official
+  DOCKER_ORG: pytorch
+  USE_BUILDX: 1
+  NO_BUILD_SUFFIX: True
+  # WITH_PUSH: ${{ github.event_name == 'schedule' }}
+
+jobs:
+  build:
+    runs-on: [self-hosted, linux.2xlarge]
+    timeout-minutes: 240
+    strategy:
+      matrix:
+        include:
+          # nvidia specific images don't exist for arm64 so only build the runtime image
```

Comment:
should this say "only build the runtime and devel images"?

Reply:
Oh this is specific to arm64

Reply:
Would it be accurate to say "# nvidia specific images don't exist for arm64 so only build the runtime image for them and not the devel image"?